.vscode/deno_resolve_npm_imports.json

```
@@ -5,7 +5,7 @@
   "// Dependency management is done through `npm` and `node_modules/` instead.": "",
   "// Deno-only dependencies may be imported via URL imports (without using import maps).": "",
   "imports": {
-    "@remix-run/server-runtime": "https://esm.sh/@remix-run/server-runtime@0.0.0-experimental-aed5bfbf7",
+    "@remix-run/server-runtime": "https://esm.sh/@remix-run/server-runtime@1.6.4",
```

We should bump this to `1.7.2` as well

```suggestion
    "@remix-run/server-runtime": "https://esm.sh/@remix-run/server-runtime@1.7.2",
```

---

yeah these don't get updated during release anymore 🥴